### DISCUSSION TITLE
GoPro Max unstable since labs firmware upgrade

### DISCUSSION THREAD

**Discussion by fxstein** [gopro/labs#123] [General]
Hi,

Ever since we upgraded our Max to the latest labs firmware we are dealing with tons of lockups. This was one of the most stable cameras before but now locks up as quickly as 30 mins into any session. 

Might have made the mistake to put a QR code intended for Hero 10 & 11s in front of it that contained the GPS Sync feature which I believe is not supported on the Max. Could that cause this?

Does anybody else experience lockups of a Max camera after installing the labs firmware?

Thanks!

**Comment by dnewman-gpsw** [comment]
Labs should not have this effect, it is supposed to be 99% passive until a feature is activated.  Only the QR Code detection is active.  I only have Labs on my Max cameras, and I have not seen any reliability issues.  The metadata style codes like !MSYNC=1, would be ignored, but stored.  So it is harmless. You can clear all stored Labs commands with !RESET.  Also factory reset for any camera that in behaving weirdly is a good idea.

**Reply by fxstein** [reply]
Very much appropriated! Will try both. 
Had so many issues lately with the Max. Even 10 min video segments where the video would freeze seconds into the shoot but audio kept going for the entire clip. After it the camera would freeze up.
